control/tap-ctl-ipc.c

```diff
@@ -187,6 +187,13 @@ tap_ctl_connect(const char *name, int *sfd)
 
 	memset(&saddr, 0, sizeof(saddr));
 	saddr.sun_family = AF_UNIX;
+
+	if (unlikely(strlen(name) > sizeof(saddr.sun_path) - 1)) {
```

Comment:
This line should be more readable by adding parenthesis around sizeof and -1 instead of relying on operator precedence.

I do see your point in raising an error instead of truncating and keep going